kikuchipy/signals/ebsd_master_pattern.py

```diff
@@ -61,6 +74,145 @@ def __init__(self, *args, **kwargs):
         self.projection = kwargs.pop("projection", None)
         self.hemisphere = kwargs.pop("hemisphere", None)
 
+    def get_patterns(
+        self,
+        rotations: Rotation,
+        detector: EBSDDetector,
+        energy: int,
+        n_chunk: Optional[int] = -1,
+        dtype_out: Optional[np.dtype] = np.float32,
+        compute: Optional[bool] = False,
+    ) -> Union[EBSD, LazyEBSD]:
+        """Creates a dictionary of EBSD patterns for a sample in the
+        (RD, TD, ND) reference frame, given a set of local crystal lattice
+        rotations and a detector model from a master pattern in the Lambert
+        projection.
+
+        Parameters
+        ----------
+        rotations : Rotation
+            Set of unit cell rotations to get patterns from.
+        detector : EBSDDetector
+            EBSDDetector object describing the detector geometry with one
+            projection center.
+        energy : int
+            The wanted energy in the master pattern.
+        n_chunk : int, optional
+            The number of chunks the data should be split up into. By default,
+            this is set so each chunk is around 100 MB.
+        dtype_out : numpy.dtype, optional
+            Data type of the returned patterns, by default np.float32.
+        compute : bool, optional
+            Wheter or not the dask.compute() function should be called, by
```

Comment:
* Whether

* I think the description should convey to the user that with `compute=True`, the patterns will be read into memory.

* I also think :func:\`dask.array.Array.compute\` should work as a reference, instead of the URL?